common/lib/pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = openedx.tests.settings
-addopts = --nomigrations --reuse-db --durations=20 --json-report --json-report-omit keywords streams collectors log traceback tests --json-report-file=none
+addopts = --nomigrations --reuse-db --durations=20 --json-report --json-report-omit keywords streams collectors log traceback tests --json-report-file=none  # lint-amnesty, pylint: disable=syntax-error
```

Comment:
This change was only added for testing a failure scenario in the second commit. This will be dropped before the merge.